superset/utils/retries.py

```diff
@@ -26,13 +27,15 @@ def retry_call(
     *args: Any,
     strategy: Callable[..., Generator[int, None, None]] = backoff.constant,
     exception: type[Exception] = Exception,
+    giveup_log_level: int = logging.WARNING,
```

Comment:
maybe type this as optional?

Reply:
I think just `int` is more precise here, since the default `logging.WARNING` is an int, and it wouldn't make sense to pass `giveup_log_level=None`